odxtools/progcode.py

```diff
@@ -34,9 +34,10 @@ def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment]) ->
         encryption = et_element.findtext("ENCRYPTION")
 
         syntax_str = odxrequire(et_element.findtext("SYNTAX"))
-        if syntax_str not in ProgCodeSyntax.__members__:
+        try:
+            syntax = ProgCodeSyntax(syntax_str)
+        except ValueError:
             odxraise(f"Encountered unknown program code syntax '{syntax_str}'")
```

Comment:
You need to add `syntax = None` inside except, otherwise the lines below will raise `NameError`

Same for other changes in this commit

Reply:
right. next try: df6db1118c9c947e0accce14eb968667a678ea53